src/routes/guides/signatures/+page.svelte

```diff
@@ -0,0 +1,157 @@
+<script lang="ts">
+  import SectionTitle from "$lib/components/SectionTitle.svelte"
+  import Code from "$lib/components/Code.svelte"
+  import json from "svelte-highlight/languages/json"
+  import A from "$lib/components/A.svelte"
+  import Breadcrumbs from "$lib/components/Breadcrumbs.svelte"
+  import Title from "$lib/components/Title.svelte"
+  import TableOfContents from "$lib/components/TableOfContents.svelte"
+  /** @type {import('./$types').PageData} */
+  export let data
```

Comment:
PyCharm highlights this like this:

![image](https://github.com/user-attachments/assets/33569e4f-426c-4c4c-9d02-d3afcc4d4c1c)

Here's what it looks like in another guide, where PyCharm is not making this complaint:
![image](https://github.com/user-attachments/assets/7ea7e056-517a-4d93-8398-bb3899989204)

Could you check if you can use that same approach and get the IDEs to not complain about it?